app/src/main/res/drawable/ksr_button_state.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <item android:color="@color/green_alpha_42"
```

Comment:
@dannyalright, is this the correct color for the disabled button states? It looks weird when the reward is long because it's transparent.
<img src=https://user-images.githubusercontent.com/1289295/59052082-45260880-885c-11e9-9748-f0e9a90c346e.png width=330/>

Reply:
It's meant to be opaque—just a lighter shade of green. That's my bad for using opacity in the design mock ups. Seeing that we don't have a green tint in our library for this, could you please add and use #57bfa4? That's our core green (#009E74) at 66% opacity. Has as much contrast as possible whilst still looking disabled.